src/documentation/components/card/card.css

```diff
@@ -0,0 +1,135 @@
+@import "../../../index.css";
+@import "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css";
+
+.checked {
+  color: var(--primary);
+}
+
+
+.card-rating {
+  margin: 0.1rem 0rem 0.2rem 0rem;
+}
+
+.card {
+  padding: 1.5rem 1.5rem;
+  background-color: var(--white);
+
+  width: 13rem;
```

Comment:
1. It is commendable that you have used rems instead of ems or px for sizing which scales reliably